docs/cli_workflow.md

```diff
@@ -9,25 +9,33 @@ name: micro-bosh
 
 networks:
 - name: default
-  type: dynamic
+  type: manual
   cloud_properties:
-    subnet: AWS_SUBNET_NAME
+    subnet: subnet-12345
+    range: 10.10.0.40/30
```

Comment:
I dont believe you need to put that (range, reserved, static) into cloud_properties. cloud_properties are evaled by the CPIs which does not care about these details.